packages/console/src/pages/Users/components/ApplicationName/index.tsx

```diff
@@ -7,9 +7,13 @@ type Props = {
 };
 
 const ApplicationName = ({ applicationId }: Props) => {
-  const { data } = useSWR<Application>(`/api/applications/${applicationId}`);
+  const isAdminConsole = applicationId === adminConsoleApplicationId;
 
-  return <span>{data?.name ?? '-'}</span>;
+  const { data } = useSWR<Application>(!isAdminConsole && `/api/applications/${applicationId}`);
+
+  const name = isAdminConsole ? 'Admin Console' : data?.name;
+
+  return <span>{name ?? '-'}</span>;
```

Comment:
Better use `||` instead of `??`, in case the name is an empty string (in theory).

Reply:
updated